### PR TITLE
Araneus is no longer a var-editted bat, and is the spiderslayer

### PR DESCRIPTION
**Pull request by ghost**
:cl: Armhulen
add: Araneus has finally gone to the gym and beefed up. Watch out, lesser spiders.
/:cl:

The hos's pet was a heavily var-editted bat and it would only run into more problems as we continue to develop the game. So, I made him an actual spider, and gave the spider some special stuff to enjoy.

I don't think this should be closed even though the bot will insist because i'm also fixing some deep rooted problems with the pet being a bat as well.

## OLD: Sergeant Araneus
does low damage, has really low health
has the tankiest spider's skin yet like i said above doesn't have really a lot of health
## NEW: Not Sergeant Araneus (but will be very soon, DWARSI!!)
Hunter variant, that has no venom but does increased damage to spiders and heals slightly when attacking them
New sprite, of which isn't in yet but get hyped for that.

### CODE REVIEW

**Comment by Repukan** [comment]
>Changing the name

Boo

**Comment by Armhulen** [comment]
You can't have a plaque without a tragic loss

**Review by ChangelingRain — APPROVED** [review]
thou must mapmerge

**Comment by Repukan** [comment]
This loss sucks.

**Comment by Armhulen** [comment]
i think that's the point

**Comment by Armhulen** [comment]
and i really didn't have a better excuse to add a spider hunting spider that looks completely different from araneus without doing this

**Comment by Repukan** [comment]
Just add it that Araneus drank 4 scoops, hit the gym and left spider kind behind

**Comment by Armhulen** [comment]
Fine

**Comment by ninjanomnom** [comment]
I agree, the old name is superior

**Comment by Armhulen** [comment]
yeah i kinda followed through with that

File: code/modules/mob/living/simple_animal/hostile/giant_spider.dm [hunk 1]
```diff
@@ -136,6 +136,23 @@
...
+	desc = "An extremely rare variant of spider found one day chomping on Araneus, He prefers spider meat moreso than human meat so we kept him around in Araneus's place."
+	icon_state = "hunter"
+	icon_living = "hunter"
+	icon_dead = "hunter_dead"
+	poison_per_bite = 0
+	move_to_delay = 5
+
+/mob/living/simple_animal/hostile/poison/giant_spider/hunter/hos/AttackingTarget()
+	. = ..()
+	if(. && isliving(target) && /mob/living/simple_animal/hostile/poison/giant_spider)
```
Comment (Qustinnus) [inline code/modules/mob/living/simple_animal/hostile/giant_spider.dm]: does `&& /mob/living/simple_animal/hostile/poison/giant_spider)` just work or what? where's the is_type()

Comment (lzimann) [reply code/modules/mob/living/simple_animal/hostile/giant_spider.dm]: no, it doesn't work, what you need to do is
`if(. && istype(target, /mob/living/simple_animal/hostile/poison/giant_spider))` and typecast accordingly

Comment (Qustinnus) [reply code/modules/mob/living/simple_animal/hostile/giant_spider.dm]: ok cool i thought i was just going crazy

**Review by ShizCalev — CHANGES_REQUESTED** [review]
Mapmerge please!

Also make the plaque a subtype too. Would love to use it somewhere else.

File: code/modules/mob/living/simple_animal/hostile/giant_spider.dm [hunk 2]
```diff
@@ -136,6 +136,23 @@
 	poison_per_bite = 5
 	move_to_delay = 5
 
+//variant of the hunter granted as the hos's pet, it has no poison but deals more damage to other spiders and heals when attacking them.
+/mob/living/simple_animal/hostile/poison/giant_spider/hunter/hos
+	name = "Commander Chorizopes"
+	desc = "An extremely rare variant of spider found one day chomping on Araneus, He prefers spider meat moreso than human meat so we kept him around in Araneus's place."
```
Comment (DoctoreMad) [inline code/modules/mob/living/simple_animal/hostile/giant_spider.dm]: Tragedy in waiting.

**Comment by Armhulen** [comment]
okay readd the plaque and i dunno what mapmerge is so someone explain that on coderbus later i suppose

File: code/modules/mob/living/simple_animal/hostile/giant_spider.dm (hunk 1 above)
Comment (Armhulen) [reply code/modules/mob/living/simple_animal/hostile/giant_spider.dm]: in the end, i was the only one going crazy

**Comment by Armhulen** [comment]
zzz sorry this'll be a bit

**Comment by Qustinnus** [comment]
if you need help with mapmerging ping me on discord :)

**Comment by Armhulen** [comment]
i'm gonna redo this pr it'll be like 2 seconds

**Pull request closed without merging**